parser/unparser_test.go

```diff
@@ -138,6 +140,191 @@ func TestUnparse(t *testing.T) {
+			out:  "a ?\nb : (c ?\nd : e)",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Conditional),
+			},
+		},
+		{
+			name: "call_wrap_or",
+			in:   "a || b || c || d || e",
+			out:  "a ||\nb ||\nc ||\nd ||\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.LogicalOr),
+			},
+		},
+		{
+			name: "call_wrap_equals",
+			in:   "a == b == c == d == e",
+			out:  "a ==\nb ==\nc ==\nd ==\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Equals),
+			},
+		},
+		{
+			name: "call_wrap_greater",
+			in:   "a > b > c > d > e",
+			out:  "a >\nb >\nc >\nd >\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Greater),
+			},
+		},
+		{
+			name: "call_wrap_greater_equals",
+			in:   "a >= b >= c >= d >= e",
+			out:  "a >=\nb >=\nc >=\nd >=\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.GreaterEquals),
+			},
+		},
+		{
+			name: "call_wrap_in",
+			in:   "a in b in c in d in e",
+			out:  "a in\nb in\nc in\nd in\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.In),
+			},
+		},
+		{
+			name: "call_wrap_less",
+			in:   "a < b < c < d < e",
+			out:  "a <\nb <\nc <\nd <\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Less),
+			},
+		},
+		{
+			name: "call_wrap_less_equals",
+			in:   "a <= b <= c <= d <= e",
+			out:  "a <=\nb <=\nc <=\nd <=\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.LessEquals),
+			},
+		},
+		{
+			name: "call_wrap_not_equals",
+			in:   "a != b != c != d != e",
+			out:  "a !=\nb !=\nc !=\nd !=\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.NotEquals),
+			},
+		},
+		{
+			name: "call_wrap_divide",
+			in:   "a / b / c / d / e",
+			out:  "a /\nb /\nc /\nd /\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Divide),
+			},
+		},
+		{
+			name: "call_wrap_modulo",
+			in:   "a % b % c % d % e",
+			out:  "a %\nb %\nc %\nd %\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Modulo),
+			},
+		},
+		{
+			name: "call_wrap_multiply",
+			in:   "a * b * c * d * e",
+			out:  "a *\nb *\nc *\nd *\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Multiply),
+			},
+		},
+		{
+			name: "call_wrap_and_long_variables",
+			in:   "longVariableA && longVariableB && longVariableC",
+			out:  "longVariableA &&\nlongVariableB &&\nlongVariableC",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.LogicalAnd),
```

Comment:
Showcase what happens when the `WrapOnOperators` is not specified. Wrapping on `&&` and `||` at 80+ chars might be a reasonable default behavior.

Reply:
[Per conversation here](https://github.com/google/cel-go/pull/559#discussion_r916352600), would supporting wrapping on blank spaces as default be a better route? It might be less surprising to the users.